search/includes/classes/class-queue.php

```diff
@@ -468,6 +468,31 @@ public function offload_indexing_to_queue() {
 		}
 	}
 
+	/**
+	 * Offload term indexing to the queue
+	 */
+	public function offload_term_indexing_to_queue( $term_id ) {
+		$term_taxonomy_id = $this->get_term_taxonomy_id( $term_id );
+
+		if ( is_null( $term_taxonomy_id ) ) {
+			return;
+		}
+
+		$this->cron->schedule_queue_posts_for_term_taxonomy_id( $term_taxonomy_id );
+	}
+
+	public function get_term_taxonomy_id( $term_id ) {
```

Comment:
This same info can be loaded via `get_term( $term_id )`, which returns a WP_Term, with a `term_taxonomy_id`: https://developer.wordpress.org/reference/classes/wp_term/

I think we should queue the cron job with just the $term_id, then let the cron job itself handle it from there.